### PR TITLE
Refactor/forms

### PR DESCRIPTION
**Pull request by taronaleksanian**

### CODE REVIEW

**Comment by andrew-bierman** [comment]
nice this is exactly what i was thinking for this feature

File: packages/app/auth/hooks/useLogin.ts [hunk 1]
```diff
@@ -8,20 +8,13 @@ interface UserForm {
...
 interface UseLoginReturn {
-  form: UseFormReturn<UserForm>;
   handleLogin: (data: UserForm) => void;
   hasError: boolean;
 }
 
 export const useLogin = (): UseLoginReturn => {
   const { mutateAsync: signIn, error } = queryTrpc.signIn.useMutation();
   const sessionSignIn = useSessionSignIn();
-  const form = useForm({
```
Comment (andrew-bierman) [inline packages/app/auth/hooks/useLogin.ts:19]: do we lose defaultValues going this route? sorry if i missed where you moved them to

Comment (taronaleksanian) [reply packages/app/auth/hooks/useLogin.ts:19]: It works perfectly even if you don't add default values, so I thought don't add them if we don't need them

File: packages/ui/src/form/components/Select.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { LmSelectRhf, type LmSelectRhfProps } from '../lib';
+
+interface Props
+  extends LmSelectRhfProps<{ label: string | number; value: any }> {}
+
+// TODO change the name to "Input" after handling tamagui all components export,
```
Comment (andrew-bierman) [inline packages/ui/src/form/components/Select.tsx:6]: ah ok i see

File: packages/ui/babel.config.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module.exports = withTamagui({
```
Comment (andrew-bierman) [inline packages/ui/babel.config.js:1]: this might need to go into expo/next folder i think, not sure the ui runs babel on build

Comment (taronaleksanian) [reply packages/ui/babel.config.js]: I bet you are right, and just fixed it

**Review by andrew-bierman — APPROVED** [review]
really nice work on this one Taron :)

**Pull request merged**